### PR TITLE
doc/functions

### PR DESCRIPTION
**Pull request by boddissattva**

### CODE REVIEW

File: docs/functions/json.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Following functions help you handle JSON documents.
+
+!!! important "The document object given to those functions could be of any type as it is always serialize as JSON string first"
```
Comment (bessonm) [inline docs/functions/json.md:3]: I think this is not enough clear

File: docs/functions/json.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+# JsonPath
+
+!!! note "Object jsonPath(Object document, String jsonPath)"
+
+    Read the given JSON path from the given document.
+
+    **Parameters** :
+
+    * `document` : The context document in which the path will be used.
```
Comment (bessonm) [inline docs/functions/json.md:14]: We may emphasize "context" means scenario execution context, or during execution

File: docs/functions/json.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    * `jsonPath` : The JSON path to read
+
+    **Returns** : The read result
```
Comment (bessonm) [inline docs/functions/json.md:17]: what if there nothing ?

**Review by bessonm — APPROVED** [review]

**Comment by bessonm** [comment]
Overall, we still need to add missing info or examples on function parameters.
Maybe we could also provide more "context" about how and why one would use a given function, to improve further understanding and to help users to have "testing" ideas and capabilities

File: docs/functions/json.md (hunk 1 above)
Comment (nbrouand) [reply docs/functions/json.md:3]: Add : "json string can not be null or empty"
All functions accept an input parameter `json` of any type.

File: docs/functions/json.md (hunk 2 above)
Comment (nbrouand) [reply docs/functions/json.md:14]: => json

File: docs/functions/json.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+    **Examples** :
+
+    SpEL : `${#jsonPath(#anyObj, '$.key[0]')}`
```
Comment (nbrouand) [inline docs/functions/json.md:21]: anyobject => json

File: docs/functions/json.md (hunk 3 above)
Comment (nbrouand) [reply docs/functions/json.md:17]: if path not found => emptyList()

**Review by nbrouand — DISMISSED** [review]

**Comment by nbrouand** [comment]
Add **parameters** when missing

**Review by bessonm — APPROVED** [review]

**Pull request merged**